corehq/apps/app_manager/management/commands/benchmark_build_times.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+from datetime import datetime
+
+from django.core.management import BaseCommand
+
+from corehq.apps.app_manager.dbaccessors import get_app
+from corehq.apps.users.models import WebUser
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'domain_app_id_pairs',
+            type=json.loads,
```

Comment:
What do you think of adding a `help="..."` parameter here with an explanation of this argument? Specifically, that it must be valid JSON, and maybe the structure of that JSON.